addons/binding/org.openhab.binding.deconz/ESH-INF/thing/thing-types.xml

```diff
@@ -136,6 +139,27 @@
 		<state readOnly="true" pattern="%.1f %unit%"></state>
 	</channel-type>
 
+	<channel-type id="light_level" advanced="true">
+		<item-type>Number</item-type>
+		<label>Light Level</label>
+		<description>Current light level.</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+
+	<channel-type id="dark">
```

Comment:
I think I do not understand the last commit. We now have "dark", "daylight", "light_level", "lightlux"?

Reply:
Those four channels are present for a light sensor (see https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Supported-Devices#measured-values-legend). Three of them are part of the Philips Hue API too. Lux is missing. From Philips Hue perspective: `dark` and `daylight` are calculated flags based on two user-defined thresholds. In the Hue APP you are able to set those values, the lower one defines threshold for darkness, the upper one defines threshold for daylight. In-between is "basic" / "indoor" illumination. Philips is working with lightlevel, whereas others prefer lux. From my point of view the lightlevel is useless thus it is advanced.

Reply:
k, thanks for the explanation